components/kdf/Pbkdf2Visualizer.tsx

```
@@ -1,161 +1,1031 @@
+          <div>
+            <p className="text-xs font-bold uppercase tracking-[0.16em] text-teal-600 dark:text-teal-400">
+              Mathematical micro-trace
+            </p>
+
+            <h2 className="mt-1 text-xl font-bold text-zinc-900 dark:text-white">
+              HMAC chaining & XOR accumulator
+            </h2>
+
+            <p className="mt-1 max-w-3xl text-sm leading-6 text-zinc-500 dark:text-zinc-400">
+              Follow the first five PBKDF2 rounds. Each HMAC output becomes
+              the next round's input, while every U value is XORed into the
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)